common/scala/src/main/scala/org/apache/openwhisk/core/connector/Message.scala

```diff
@@ -636,6 +636,23 @@ object ContainerMessage extends DefaultJsonProtocol {
 sealed trait ContainerCreationError
 
 object ContainerCreationError extends Enumeration {
+  import scala.language.implicitConversions
+  implicit def containerCreationErrorToString(x: ContainerCreationError): String = {
```

Comment:
I added an implicit conversion to remove the `reason` field from this.
https://github.com/apache/openwhisk/pull/5313/files#diff-32b47fe5e4d4cdb36b4ea9095511f467e626667c207ca52f3064535d4b8b28d3R56